packages/@expo/cli/src/prebuild/resolveLocalTemplate.ts

```diff
@@ -1,11 +1,23 @@
 import type { ExpoConfig } from '@expo/config';
 import fs from 'fs';
+import path from 'path';
 import resolveFrom from 'resolve-from';
 
-import { extractNpmTarballAsync } from '../utils/npm';
+import { copyNodeModuleAsync, extractNpmTarballAsync } from '../utils/npm';
 
 const debug = require('debug')('expo:prebuild:resolveLocalTemplate') as typeof console.log;
 
+/** Returns the `local-template` target path, only for the `expo/expo` monorepo */
+const getMonorepoTemplatePath = async () => {
+  const cliPath = path.dirname(require.resolve('@expo/cli/package.json'));
+  const localTemplateOriginPath = path.join(cliPath, 'local-template');
```

Comment:
i would lean toward to use `__dirname/../../...` than resolving @expo/cli/package.json again. think the case where people in a project outside of expo/expo and execute expo-cli from expo/expo. maybe good to keep behavior consistent

```sh
$ cd sdk54
$ /path/to/expo/expo/packages/@expo/cli/build/bin/cli prebuild
```